libs/angular-generator/src/lib/theming/theming-service.ts

```diff
@@ -0,0 +1,71 @@
+import { ExportSpecifierType, GeneratedFile } from "@ui5/webcomponents-wrapper";
+import { format } from "prettier";
+
+export class Ui5GlobalThemingService extends GeneratedFile {
+  constructor() {
+    super();
+    this.move('ui5-theming.service.ts');
+    this.addExport({
+      local: 'Ui5ThemingService',
+      exported: 'Ui5ThemingService',
+      types: [ExportSpecifierType.Class],
+    });
+  }
+
+  override getCode(): string {
+    this.addImport(['Inject', 'Injectable', 'isDevMode'], '@angular/core');
+    this.addImport(['AvailableThemes', 'ThemingConfig', 'UI5_THEMING_CONFIGURATION', 'Ui5ThemingProvider', 'Ui5ThemingConsumer'], './ui5-theming.models');
```

Comment:
I would make second parameter more dynamic. If user renames `ui5-theming.models` file name, this code will not be able to detect that on generation